feg/gateway/docker/docker-compose.yml

```diff
@@ -40,51 +40,64 @@ x-goservice: &goservice
   <<: *service
   image: ${DOCKER_REGISTRY}gateway_go:${IMAGE_VERSION}
 
+x-log-forward: &depends_on_logs
+  depends_on:
+    td-agent-bit:
+      condition: service_healthy
+
 services:
   csfb:
     <<: *goservice
+    <<: *depends_on_logs
```

Comment:
yeah we may try but  `td-agent-bit ` also uses `<<: *pyservice` which inherits  `&service`  so we may end up with container which depends on itself.

Reply:
what about creating this so `td-agent-bit` uses `pyservice_base` and the other services use `pyservice`?


```
# Use generic python anchor to avoid repetition for python services
x-pyservice_base: &pyservice_base
  <<: *service
  image: ${DOCKER_REGISTRY}gateway_python:${IMAGE_VERSION}
  
# Use generic python anchor to avoid repetition for python services
x-pyservice: &pyservice
  <<: *pyservice_base
  <<: *depends_on_logs
  image: ${DOCKER_REGISTRY}gateway_python:${IMAGE_VERSION}

```

Reply:
yeah it makes sense for me let me do it